.github/workflows/benchmarks.yml

```
@@ -4,39 +4,44 @@ on:
   workflow_dispatch:
     inputs:
       runStandalone:
-        description: 'Run the benchmarks against standalone APM Server with Moxy'
+        description: 'Benchmark against standalone APM Server with Moxy'
         required: false
         type: boolean
         default: false
       enableTailSampling:
-        description: 'Enable tail-based sampling on the APM server'
+        description: 'Enable tail-based sampling on APM server'
         required: false
         type: boolean
         default: false
       tailSamplingStorageLimit:
-        description: 'Storage size limit of tail-based sampling on the APM server, defaults to 10GB'
+        description: 'Storage size limit of tail-based sampling on APM server'
         required: false
         type: string
         default: "10GB"
       profile:
-        description: 'The system profile used to run the benchmarks'
+        description: 'System profile for benchmark'
         required: false
         type: string
+        default: "system-profiles/8GBx1zone.tfvars"
       runOnStable:
-        description: 'Run the benchmarks on the latest stable version'
+        description: 'Benchmark on latest stable version'
         required: false
         type: boolean
         default: false
       benchmarkAgents:
-        description: 'Set the number of agents to send data to the APM Server'
+        description: 'Number of benchmark agents sending data to APM Server'
         required: false
         type: string
       benchmarkRun:
-        description: 'Set the expression that matches the benchmark scenarios to run'
+        description: 'Expression for benchmark scenarios to run'
+        required: false
+        type: string
+      warmupTime:
+        description: 'Benchmark warmup time for APM Server'
```

```suggestion
        description: 'Benchmark warmup time for APM Server, e.g. 5m'
```
On the GH Actions UI, it is unclear what's the format for the warmup time input. People with a Go background _might_ assume that it takes a time unit, while Python people _might_ assume it is in seconds.